docker/dev/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Engelsystem PHP FPM development image including Xdebug
+FROM php:7.3-fpm-alpine3.10 AS php_fpm
+WORKDIR /var/www
+RUN apk add --no-cache icu-dev $PHPIZE_DEPS && \
+    pecl install xdebug && \
+    docker-php-ext-install intl pdo_mysql && \
+    docker-php-ext-enable xdebug
+COPY ./docker/dev/php/xdebug.ini /usr/local/etc/php/conf.d
```

Comment:
As long as we don't do more advanced stuff i would just echo the contents to the file to prevent even more useless files laying around?

Reply:
Echo in place ✓